kernel/kernel-default/src/test/scala/io/delta/kernel/DeltaTableReadsSuite.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (2021) The Delta Lake Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.delta.kernel
+
+import java.sql.Timestamp
+
+import io.delta.golden.GoldenTableUtils.goldenTablePath
+import org.apache.hadoop.conf.Configuration
+import org.scalatest.funsuite.AnyFunSuite
+
+class DeltaTableReadsSuite extends AnyFunSuite with TestUtils {
```

Comment:
I added these tests in scala just since it's so much easier. I could rewrite them into the java suite if needed. But I think it is easier for all our testing to be in scala (and at some point we should move over the java tests.)